test/unit/Chainweb/Test/Pact5/PactServiceTest.hs

```diff
@@ -467,21 +449,18 @@ tests = do
     -- * test that rewinding past the pact5 boundary is possible
 -}
 
-cid :: ChainId
-cid = unsafeChainId 0
+chain0 :: ChainId
```

Comment:
this is so obvious in hindsight lol

Reply:
Right? This treatment should move to other suites too.